src/Sulu/Bundle/ContactBundle/Resources/public/js/components/accounts/components/content/main.js

```diff
@@ -17,7 +17,12 @@ define(function() {
                 url: '/admin/content-navigations?alias=account'
             },
             toolbar: {
-                template: 'default'
+                buttons: [
+                    'saveWithOptions',
+                    {'settings': {
+                        dropdownItems: ['delete']
+                    }}
```

Comment:
I would go for `parent`, since all the properties are "inherited". This makes also sense, since I guess.

```javascript
buttons: {
    saveWithOptionsSpecial: {
        parent: 'saveWithOptions'
    },
    settings: {
        options: {
            dropdownItems: {
                delete: {
                    parent: 'delete'
                }
            }
        }
    }
}
```

The attribute name (`settings`, `saveWithOptions`) would be the `id` to use for emitting events, ...